examples/connect5.rs

```diff
@@ -556,7 +555,7 @@ fn search(pos: &Pos, alpha: i32, beta: i32, depth: i32, _ply: i32) -> i32 {
     }
 
     assert_ne!(bm, MOVE_NONE);
-    assert!(bs >= -EVAL_INF && bs <= EVAL_INF);
+    assert!((-EVAL_INF..=EVAL_INF).contains(&bs));
```

Comment:
I think this is actually less clear than the previous code. I'd like this specific line to be reverted.